src/09-01.zig

```diff
@@ -0,0 +1,14 @@
+const std = @import("std");
+const assert = std.debug.assert;
+
+pub fn main() !void {
+    const version = try std.SemanticVersion.parse("0.2.6");
+
+    assert(version.order(.{
```

Comment:
Why not use `version == .{}` directly?

Reply:
Here is the error message.
```
error: operator == not allowed for type 'SemanticVersion'
    assert(version == std.SemanticVersion{
```
Need to compare using the `order` method.